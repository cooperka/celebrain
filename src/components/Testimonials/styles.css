.Testimonials {
  min-height: 30rem;
  margin: 4rem 0;
  display: flex;
  flex-direction: column;
  font-weight: 500;
}

.Testimonials .title {
  margin: 1rem;
  font-size: 5rem;
}

.Testimonials .section {
  margin: 1rem auto;
}

.Testimonials .subtitle {
  margin: 1rem auto;
  font-size: 2.2rem;
  max-width: 44rem;
}
