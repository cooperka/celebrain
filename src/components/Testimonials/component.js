// @flow

import React from 'react';

import './styles.css';

function Testimonials() {
  return (
    <div className="Testimonials">
      <div className="title">Testimonials</div>
    </div>
  );
}

export default Testimonials;
