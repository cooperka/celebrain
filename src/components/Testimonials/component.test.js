// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Testimonials from './component';

it('renders', () => {
  const tree = renderer.create(<Testimonials />);
  expect(tree.toJSON()).toMatchSnapshot();
});
