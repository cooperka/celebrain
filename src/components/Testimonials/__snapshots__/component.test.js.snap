// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="Testimonials"
>
  <div
    className="title"
  >
    Testimonials
  </div>
</div>
`;
